src/helper.js

```diff
@@ -67,6 +67,12 @@ function setHome(homePath) {
 		var userConfig = require(this.CONFIG_PATH);
 		this.config = _.extend(this.config, userConfig);
 	}
+
+	// TODO: Remove in future release
+	if (this.config.debug === true) {
+		log.warn("debug option is now an object, see defaults file for more information.");
+		this.config.debug = {ircFramework: true};
+	}
```

Comment:
If we continue to do this, maybe we should do it in the `Helper` instead, so all warnings show up together (well, they probably do right now, but at least they'd be together in the code). No big deal here though.